gen/xyz/openbmc_project/LID/Common/meson.build

```diff
@@ -0,0 +1,14 @@
+# Generated file; do not modify.
```

Comment:
Any reason we can't send this upstream?

Reply:
I think update access key verification was proposed to the community but it got rejected, and since this interface is mainly for that, it was suggested that it be setup downstream

Reply:
Yeah, Brad mentioned we add this logic downstream either as a new d-bus method or have the downstream oem pldm logic that handles inband update to this verification by duplicating the logic in the bmc updater code, so we're going with the a new d-bus method.